projects/Mallard/src/helpers/push-tracking.ts

```diff
@@ -72,10 +74,12 @@ const pushTracking = async (
         }
 
         const storedTracking = await AsyncStorage.getItem(PUSH_TRACKING_KEY)
+        const { type } = await NetInfo.fetch()
```

Comment:
you're doing this directly rather than via apollo - is this deliberate?

Reply:
As its not in a component, we cant use the hook so we need to use Netinfo directly. One of the reasons to use the hook is to mock Netinfo to easier test different network connections. But Push Tracking is supplementary to the experience so I think in this case its ok.